pymc_marketing/mmm/utils.py

```diff
@@ -33,3 +34,95 @@ def generate_fourier_modes(
             for func in ("sin", "cos")
         }
     )
+
+def find_elbow(x: np.array, y: np.array) -> int:
```

Comment:
I think we should use the hints as  in https://github.com/pymc-labs/pymc-marketing/actions/runs/5017993802/jobs/8997616662?pr=287. In this case `x:  npt.NDArray[np.float_]`. Otherwise `mypy` complains 
```python
pymc_marketing/mmm/utils.py:39: error: Function "numpy.core.multiarray.array" is not valid as a type  [valid-type]
```

Reply:
Same for all methods 😅